README.md

```diff
@@ -225,7 +225,12 @@ There is currently only one GUC parameter to enable/disable the `pg_parquet`:
 | `crunchy_map`(5)  | GROUP                     | MAP              |
 
 > [!WARNING]
-> - (1) The `numeric` types with <= `38` precision is represented as `FIXED_LEN_BYTE_ARRAY(16)` with `DECIMAL(128)` logical type. The `numeric` types with > `38` precision is represented as `BYTE_ARRAY` with `STRING` logical type.
+> - (1) `numeric` type is written the smallest possible memory width to parquet file as follows:
+>    * `numeric(P <= 9, S)` is represented as `INT32` with `DECIMAL` logical type
+>    * `numeric(9 < P <= 18, S)` is represented as `INT64` with `DECIMAL` logical type
+>    * `numeric(18 < P <= 38, S)` is represented as `FIXED_LEN_BYTE_ARRAY(9-16)` with `DECIMAL` logical type
+>    * `numeric(38 < P, S)` is represented as `BYTE_ARRAY` with `STRING` logical type
+>    * `numeric` is allowed by Postgres. (precision and scale not specified). These are represented by a default precision (38) and scale (16) instead of writing them as string. You get runtime error if your table tries to read or write a numeric value which is not allowed by the default precision and scale (22 integral digits before decimal point, 16 digits after decimal point).
```

Comment:
I believe this is true, but just being explicit in my understanding: So effectively if you want `numeric` with larger precision than 38 you need to explicitly define it there, it just will not end up being an actual stored numeric, but reads/writes will work to convert transparently from the Postgres side?

Reply:
True. An example

```sql
pg_parquet=# create table test(x numeric(40,2));
CREATE TABLE

pg_parquet=# insert into test values (1.23);
INSERT 0 1

pg_parquet=# copy test to '/tmp/test.parquet';
COPY 1

pg_parquet=# select * from parquet.metadata('/tmp/test.parquet');
-[ RECORD 1 ]-----------+-------------------------
uri                     | /tmp/test.parquet
row_group_id            | 0
row_group_num_rows      | 1
row_group_num_columns   | 1
row_group_bytes         | 65
column_id               | 0
file_offset             | 0
num_values              | 1
path_in_schema          | x
type_name               | BYTE_ARRAY
stats_null_count        | 0
stats_distinct_count    | 
stats_min               | 1.23
stats_max               | 1.23
compression             | SNAPPY
encodings               | PLAIN,RLE,RLE_DICTIONARY
index_page_offset       | 
dictionary_page_offset  | 4
data_page_offset        | 28
total_compressed_size   | 69
total_uncompressed_size | 65

pg_parquet=# copy test from '/tmp/test.parquet';
COPY 1
```